extensions/adobe/experience/decisioning/proposition-response-details.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/response-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Response Details",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "A decision response is used to link an experience event that is a response to a prior decision/proposition events. Each decision output has a unique ID that can be echo'ed back with an experience event. The decision is assumed to have contributed to the experience event and is now attributed with by it.",
+  "definitions": {
+    "decision-response-datatype": {
+      "type": "object",
+      "properties": {
+        "xdm:decisionEventId": {
```

Comment:
@jbeckert  - I see we have namespaced this field under "xdm". Given the location and purpose of this field, should this field be named "https://ns.adobe.com/experience/decisioning/decisionEventId" so its associated with the "experience/decisioning" namespace?

This same comment would apply the the "xdm:eventContext" below.

Reply:
it is a local field inside a mix-in structure that has the namespace treatment. Have a look at the examples for a better illustration of this.

Reply:
Yes, I see this and it's something we did in the past because of the XED conversion to avoid the duplicate namespaces in the physical path. The XED conversion no longer duplicates so it's technically not necessary. We can leave it as it this time, but note in the future we need to start having vendor/solution mixins declare new fields in their own namespace. If it's an "xdm" field then that should be declared inside a standard "xdm" mixin and not one from a solution/vendor.